SharedCode/Util/StorageUtils.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using SharedCode.Interfaces;
+
+namespace SharedCode.Util
+{
+
+	public class StorageUtils: IStorageUtils
+	{
+        private IStorage Storage;
+
+        public StorageUtils(IStorage storage)
+        {
+            this.Storage = storage;
+        }
+
+        public bool GetIsLoggedIn()
+        {
+            return Storage.Read("isLoggedIn", false);
```

Comment:
Maybe define the string as a constant value would be better so it'll be easier to change in the future

Reply:
Changed.